### DISCUSSION TITLE
Mobile

### DISCUSSION THREAD

**Discussion by Anatolysco** [4tals/LinksForIsrael#347] [Ideas]
**I want to start a discussion about a brand-new design for our mobile version!**
 
It's have come up with a slick solution that'll make browsing through our myriad of categories a breeze. In addition, we can bring a geolocation search, so finding what you need based on where you are is going to be much intuitive. We're talking about a smooth content organization that give an ability to find what you're looking for with just a few taps. 

Best part part of it that we're keeping all the content you know and love right where it is. So, let's discuss about and bring this new version to life. We're not just keeping up; we're setting the pace.

1. [Demo that works both for mobile and desktop (hot spots will lead your way):
](https://www.figma.com/proto/BxHCYKopVxDdMmuIoOQ0AL/Look-and-Feel?type=design&node-id=369-29174&t=bLJfbyhbVPrsvTUt-0&scaling=min-zoom&page-id=309%3A2868&starting-point-node-id=369%3A29174&hide-ui=1)

2. [Figma (with a design mock ups)](https://www.figma.com/file/BxHCYKopVxDdMmuIoOQ0AL/Look-and-Feel?type=design&node-id=309-6197&mode=design&t=8Z5IE5w15GCNOHen-4)

**Looking forward to diving into this with you all! @4tal @yanai101 @ohadschn @oriharel 
Adds to the discussion those you consider necessary and feel free to ask questions**

![Mobile](https://github.com/4tals/LinksForIsrael/assets/12582720/43f255e9-321d-4aca-b928-8457e3e7eece)

**Comment by omer-friedman** [comment]
Consider using [FluentUI](https://developer.microsoft.com/en-us/fluentui#/controls/web/).
It will make development much easier and faster.
